crates/bevy_ecs/src/change_detection.rs

```diff
@@ -542,6 +542,45 @@ impl<'a, T: ?Sized> Ref<'a, T> {
     pub fn into_inner(self) -> &'a T {
         self.value
     }
+    /// Map `Ref` to a different type using `f`.
+    ///
+    /// Note that unlike [`Map::map_unchanged`] this doesn't do anything
```

Comment:
Nit: Did you mean this?
```suggestion
    /// Note that unlike [`Mut::map_unchanged`] this doesn't do anything
```